app/models/user.rb

```diff
@@ -0,0 +1,33 @@
+require 'bcrypt'
+
+module Armadillo
+  module Models
+    class User
+      attr_reader :password
+      attr_accessor :password_confirmation
+      include DataMapper::Resource
+
+      property :id, Serial
+      property :email, String, required: true
+      property :password_digest, Text
+      property :name, String, required: true
+      property :user_name, String, required: true
+
+      has n, :peeps, through: Resource
+
+      validates_uniqueness_of :email
```

Comment:
Prefer the new style validations `validates :column, uniqueness: value` over `validates_uniqueness_of`.